e2e/cephfs.go

```diff
@@ -427,6 +427,90 @@ var _ = Describe(cephfsType, func() {
 				}
 			})
 
+			By("verify rados objects are within a namespace", func() {
+				framework.Logf("updating configmap with rados namespace %s", radosNamespace)
+				radosNamespace = "cephfs-ns"
```

Comment:
Poeh, this is a global variable, changing it during runtime can possibly have unexpected side-effects. Can you add a comment that it is critical that the value is restored to whatever it was (maybe not `""`?) at the end of this test. 

Or, combine the changing with restoring together, like so:

```go
oldRadosNamespace := radosNamespace
radosNamespace = "cephfs-ns"
defer func() {
    radosNamespace = oldRadosNamespace
}
```